app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,11 @@
     <%#= tag.link rel: "manifest", href: pwa_manifest_path(format: :json) %>
 
     <%= favicon_link_tag %>
-    <%= stylesheet_link_tag :all, "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag :application, "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag :tailwind, media: "all", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag :trix, media: "all", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag :actiontext, media: "all", "data-turbo-track": "reload" %>
```

Comment:
I propose group sheets that are connected. There will be no difference in generated HTML, but is can be information that `trix` and `actiontext` are connected:

```suggestion
    <%= stylesheet_link_tag :trix, :actiontext, media: "all", "data-turbo-track": "reload" %>
```